src/Strategy/Node.hs

```diff
@@ -214,6 +216,15 @@ data YarnVersion
   = V1
   | V2Compatible
 
+findWorkspaceNames :: PkgJsonGraph -> WorkspacePackageNames
```

Comment:
[optional] [nit] Prefer tiny doc comment explaining how we find these in layman's terms for future code readers :)